Driverless AI/disparate-impact-analysis-tutorial/disparate-impact-analysis-tutorial.md

```diff
@@ -0,0 +1,458 @@
+# Disparate Impact Analysis Tutorial 
+
+## Outline
+- [Objective](#objective)
+- [Prerequisites](#prerequisites)
+- [Task 1: Launch Machine Learning Interpretability Experiment](#task-1:-launch-machine-learning-interpretability-experiment) 
```

Comment:
The links to the remaining tasks are broken. When linking each task you don't have to use ":"

The following tasks need to have the ":" removed in the (#task4-name-of-section) so they could work
Example:

[Task 1: Launch Machine Learning Interpretability Experiment](#task-1-launch-machine-learning-interpretability-experiment)

Task 1: Launch Machine Learning Interpretability Experiment
Task 2: Concepts
Task 3: Confusion Matrix
Task 4: Disparate Impact Analysis (DIA)
Task 5: Sensitivity Analysis Part 1
Task 6: Sensitivity Analysis Part 2

Reply:
@Castro38 thanks